tests/scripts/pulp_file/test_acs.sh

```diff
@@ -37,7 +37,12 @@ test "$(echo "$OUTPUT" | jq ".paths | length")" -eq 2
 # test refresh
 expect_succ pulp file acs refresh --name $acs
```

Comment:
maybe add `--background` here.

Reply:
Will "ERROUTPUT" be still accessible on the next line?

Reply:
It will only be overwritten on the next "expect_*"

Reply:
Or you mean that "--background" still tells you about the triggered taskgroup? I hope so.